### DISCUSSION TITLE
Cannot renew any certs at all as of 26.07.24

### DISCUSSION THREAD

**Discussion by PWorsley** [rcl-ssl/docs#47] [General]
Hi Im trying to renew two expired stand alone certificates. After Deleting, I get errors after checking them on Dig the validation phase seems to fail. Occasionally I dont get to the check on Dig phase.
 Does anyone know whats going on here ? , 

Error:Could not create Stand Alone certificate : ---.com with challenge : DNS Could not fetch nonce : Object reference not set to an instance of an object.

**Comment by rcladmin** [comment]
A recent fix was done for the 'Could Not Fetch Nonce' Error. Please try again and see if the error goes away.
